packages/xrpl/src/models/transactions/common.ts

```diff
@@ -500,7 +505,13 @@ export function validateCredentialsList(
   }
 }
 
-function containsDuplicates(objectList: object[]): boolean {
+/**
+ * Check if an array of objects contains any duplicates.
+ *
+ * @param objectList - Array of objects to check for duplicates
+ * @returns True if duplicates exist, false otherwise
+ */
+export function containsDuplicates(objectList: object[]): boolean {
   const objSet = new Set(objectList.map((obj) => JSON.stringify(obj)))
```

Comment:
This will be incorrect if one has `{Issuer: ..., CredentialType: ...}` and the other has `{CredentialType: ... Issuer: ...}`

Reply:
thanks for pointing it out. How does this look ? [da655b2](https://github.com/XRPLF/xrpl.js/pull/2874/commits/da655b20ac59172331999b5aa61ea22f62a60124)